### ISSUE TITLE
 PostageLabel create

### ISSUE THREAD

**Issue by lirazgre** [EasyPost/easypost-java#27]
Hi, 
I am trying to create a PostageLabel in order to change label size, but am getting an exception.

```
java.lang.reflect.InvocationTargetException
com.easypost.exception.EasyPostException: An error occured. Response code: 404 Response body: {"error":{"code":"NOT_FOUND","message":"The requested resource could not be found.","errors":[]}}
```
 by running:
  ```
 private PostageLabel createPostageLabel()throws EasyPostException{
        EasyPost.apiKey = API_TEST_KEY;

        Map<String, Object> labelMap = new HashMap<>();
        labelMap.put("label_size", "5x5");

        return PostageLabel.create(labelMap);
    }
```
I was wondering if it has anything to do with mapping the params to "parcel" and not "postage_label" in PostageLabel class: 

```
public static PostageLabel create(Map<String, Object> params, String apiKey) throws EasyPostException {
		Map<String, Object> wrappedParams = new HashMap<String, Object>();
		wrappedParams.put("parcel", params);
               
 return request(RequestMethod.POST, classURL(PostageLabel.class), wrappedParams, PostageLabel.class, apiKey);
	}

```

Thanks,
Liraz

**Comment by att14** [comment]
You can't directly create a `PostageLabel` object. I belive the `create` function is a bad copy paste. We will remove it in the next release.

To create a `PostageLabel` you must first create a `Shipment` and then purchase the shipment with a valid `Rate`. See the official documentation for more details: https://www.easypost.com/docs/api/java#create-a-shipment.

**Comment by lirazgre** [comment]
I did look at the API examples (they were very helpful , thanks) and  I do create a Shipment first.

I do a `Shipment shipment = Shipment.create(shipmentMap);`  
and after  `shipment.buy(shipment.lowestRate(buyCarriers, buyServices));`
I know that the postageLable is set only **after** the `buy` so I tried `setLabelSize` (before printing) . 
I might doing it wrong - but the png label (looking at `labelUrl`) doesn't seem to be effected by any of my setting changes ( I also tryied playing with `setLabelFileType`)

Thanks again

**Comment by victoryftw** [comment]
We've just released a new version of this library (v3.0.1) that removes the non-functional PostageLabel create method. It also makes a few setter methods that were Public into Private methods, as they should have been all along.

As for setting the label's size and type, it seems like what you're actually looking for are the Shipment Options "label_size" and "label_format", which you should specify at Shipment create time. You can read more about our Shipment Options on the [Shipment Options Documentation page](https://www.easypost.com/docs/api/java#options). Setting these options at create time will cause our system to return the proper label when you actually buy the postage!
